sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -95,8 +95,10 @@ def begin_purchase_phone_numbers(
             for LRO operations if no Retry-After header is present.
```

Comment:
Would be nice to take the time to document what is the unit of the polling interval e.g. seconds